examples/platform/silabs/efr32/BUILD.gn

```diff
@@ -110,6 +110,20 @@ source_set("efr32-factory-data-provider") {
     "${chip_root}/src/platform:platform_base",
     "${chip_root}/src/setup_payload",
   ]
+}
+
+config("ICD-subscription-callback-config") {
+  defines = [ "CHIP_CONFIG_USE_ICD_SUBSCRIPTION_CALLBACKS=1" ]
+}
+
+source_set("efr32-ICD-subscription-callback") {
```

Comment:
why is this source set referencing sources in the above folder? Why don't we have a source set in a BUILD.gn above?

Reply:
We are currently refactoring our build.gn to support a new platform (siWx917) and it's not complete yet.